components/create_comment/create_comment.jsx

```diff
@@ -652,15 +676,29 @@ export default class CreateComment extends React.PureComponent {
             />
         );
 
-        const notifyAllMessage = (
-            <FormattedMessage
-                id='notify_all.question'
-                defaultMessage='By using @all or @channel you are about to send notifications to {totalMembers} people. Are you sure you want to do this?'
-                values={{
-                    totalMembers: this.props.channelMembersCount - 1,
-                }}
-            />
-        );
+        let notifyAllMessage = '';
+        if (this.state.countChannelTimezones && this.props.isTimezoneEnabled) {
+            notifyAllMessage = (
+                <FormattedMarkdownMessage
+                    id='notify_all.question_timezone'
+                    defaultMessage='By using @all or @channel you are about to send notifications to **{totalMembers} people** in **{timezones, number} {timezones, plural, one {timezone} other {timezones}}**. Are you sure you want to do this?'
+                    values={{
+                        totalMembers: this.props.channelMembersCount - 1,
+                        timezones: this.state.countChannelTimezones,
```

Comment:
Thanks for addressing the feedback, @cpanato! Just realized my suggestion for `countChannelTimezones` would have been better as `channelTimezonesCount` to match the incoming `channelMembersCount` prop. 0/5, and not blocking :)